contracts/cw20-merkle-airdrop/helpers/src/airdrop.ts

```diff
@@ -18,23 +18,22 @@ class Airdrop {
     amount: string;
   }): string[] {
     return this.tree
-      .getHexProof(keccak256(account.address + account.amount))
+      .getHexProof(sha256(account.address + account.amount).toString())
       .map((v) => v.replace('0x', ''));
   }
 
   public verify(
     proof: string[],
     account: { address: string; amount: string }
   ): boolean {
-    let hashBuf = keccak256(account.address + account.amount);
+    let hashBuf = Buffer.from(sha256(account.address + account.amount).toString())
 
     proof.forEach((proofElem) => {
       const proofBuf = Buffer.from(proofElem, 'hex');
-
       if (hashBuf < proofBuf) {
-        hashBuf = keccak256(Buffer.concat([hashBuf, proofBuf]));
+        hashBuf = Buffer.from(sha256(Buffer.concat([hashBuf, proofBuf]).toString()));
```

Comment:
why type does sha256 return? Why `Buffer.from(sha256(a).toString())`?
Seems like there may be an easier way to transform it.

(Also if using Buffer.from, I think best to specify the input type. I think the default is `hex`, but let's be specific)

Reply:
sha256 returns `CryptoJS/WordArray` which is hard to work with. Only `toString` method available.

Reply:
Well, best you could do with what you got.